.github/workflows/release.yml

```diff
@@ -118,38 +117,12 @@ jobs:
```

Comment:
- In the `Append Fix Notes to Confluence` step, the conversion of `new_fix_notes` to Confluence format using sed might not cover all edge cases. Ensure robust handling of formats.